content/applications/finance/payment_acquirers.rst

```diff
@@ -169,6 +169,12 @@ authenticate the payment, or if to stay on the current page and authenticate the
 .. note::
    Some features are available only if you select *Redirection to the acquirer website*.
 
+.. warning::
+   When configuring a payment acquirer with the payment flow :guilabel:`Payment from Odoo`, the
+   payment acquirer is visible only to people who have an account on the `Odoo.com <https://www.odoo
```

Comment:
Would it be clearer to simply say "users logged into their account"?

Reply:
If I understood correctly, users who do not have an account on the Odoo.com portal cannot see the payment acquirer, including both non-logged users and users who do not have an account. I find "users logged into their account" implies the doc/warning only concerns users having an Odoo account not logged in, and not those who do not have an account at all.

Reply:
Oh, it's actually totally unrelated to the odoo.com database! I didn't read that well. The only thing that matters is whether the user is logged in to their (internal or portal) account on the merchant's database (i.e. the one they're trying to make a payment to).

Reply:
Done!